### PR TITLE
(PC-20784) chore(env): update reco api URLs

### PR DESCRIPTION
**Pull request by voisinhugo**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-20784

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: .env.integration [hunk 1]
```diff
@@ -85,5 +85,6 @@ AMPLITUDE_API_KEY=
 
 # Stores
 APPLE_STORE_URL=https://apps.apple.com/fr/app/pass-culture/id1557887412
-GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp&hl=fr
+GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp &
+hl=fr
```
Comment (anoukhello) [inline .env.integration:89]: C'est normal ça ?

File: .env.staging [hunk 2]
```diff
@@ -112,5 +112,6 @@ AMPLITUDE_API_KEY=e92915bd645d6bb5b044102d1ed5933e
 
 # Stores
 APPLE_STORE_URL=https://apps.apple.com/fr/app/pass-culture/id1557887412
-GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp&hl=fr
+GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp &
+hl=fr
```
Comment (anoukhello) [inline .env.staging:116]: idem

File: .env.testing [hunk 3]
```diff
@@ -112,5 +112,6 @@ AMPLITUDE_API_KEY=e92915bd645d6bb5b044102d1ed5933e
 
 # Stores
 APPLE_STORE_URL=https://apps.apple.com/fr/app/pass-culture/id1557887412
-GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp&hl=fr
+GOOGLE_PLAY_STORE_URL=https://play.google.com/store/apps/details?id=app.passculture.webapp &
+hl=fr
```
Comment (anoukhello) [inline .env.testing:116]: idem

File: .env.integration (hunk 1 above)
Comment (voisinhugo) [reply .env.integration:89]: Nope, mon linter qui est devenu fou, j'avais pas vu !

**Review by anoukhello — APPROVED** [review]

**Pull request merged**